openmc/data/njoy.py

```diff
@@ -361,15 +361,15 @@ def make_ace(filename, temperatures=None, ace=None, xsdir=None,
     run(commands, tapein, tapeout, **kwargs)
 
     if acer:
-        ace = os.path.join(output_dir, "ace") if ace is None else ace
-        xsdir = os.path.join(output_dir, "xsdir") if xsdir is None else xsdir
-        with open(ace, 'w') as ace_file, open(xsdir, 'w') as xsdir_file:
+        ace = (output_dir / "ace") if acer is True else Path(acer)
+        xsdir = (ace.parent / "xsdir") if xsdir is None else xsdir
+        with ace.open('w') as ace_file, xsdir.open('w') as xsdir_file:
```

Comment:
I love `pathlib` more and more by the day. _Of course_ paths have an `open` method!